src/app/attendances/page.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import TopNavBar from '../molecules/top-navigation-bar';
+import AttendanceStatus from '../molecules/attendance-status';
+import styles from './attendances.module.css';
+
+const AttendancesPage = () => {
+  const [searchParams] = useSearchParams();
+  const subjectId = searchParams.get('subject_id');
+  const [subject, setSubject] = useState(null);
+
+
+    const fetchSubjectData = async () => {
+      try {
+        const url = `/attendances/${subjectId}`;
```

Comment:
```suggestion
        const url = `${API_ROOT}/attendances/${subjectId}`;
```
これは有効な URL ではありません. API のルートの URL にあたる文字列を用意してあるので, インポートに `import { API_ROOT } from "@/queries/config";` を追加しつつこのように書いて URL を完成させましょう.